src/app/shared/components/lib/default/default.module.ts

```diff
@@ -0,0 +1,11 @@
+import { NgModule }           from '@angular/core';
+import { CommonModule }  from '@angular/common';
+
+import { DefaultComponent } from './default.component';
+
```

Comment:
`export * from ./default.component`

This will re-export the component itself so your app developers can import it and use it in their code.